imperial_coldfront_plugin/views.py

```diff
@@ -51,3 +61,93 @@ def group_members_view(request: HttpRequest, user_pk: int) -> HttpResponse:
     group_members = GroupMembership.objects.filter(owner=user)
 
     return render(request, "group_members.html", {"group_members": group_members})
+
+
+@require_POST
+@login_required
+def invite_to_group(request: HttpRequest) -> HttpResponse:
```

Comment:
We should use a Django form class as this will give us extra validation and error handling logic. The django docs on it are quite good - https://docs.djangoproject.com/en/5.1/topics/forms/#building-a-form.